frontend/components/welcomeHeader.jsx

```diff
@@ -17,27 +17,58 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
```

Comment:
don't need these unless you put some styles in them

Reply:
Should the Medina type font be thrown in here?